src/figuresClasses.ts

```diff
@@ -1,19 +1,78 @@
-export interface Figure {
+// import { getTypeParameterOwner } from 'typescript';
 
+export interface Figure {
+  shape: string;
+  color: string;
```

Comment:
you have defined values for shape and color, so create an enums for them
![image](https://user-images.githubusercontent.com/122989585/221349554-021fd8ad-7254-4ec3-b058-94484a28563a.png)